### ISSUE TITLE
Listing Tile images does not have alt and Link does not have Text

### ISSUE THREAD

**Issue by tkimnguyen** [castlecms/castle.cms#77]
Ensures <img> elements have alternate text or a role of none or presentation

wcag2a
wcag111
section508
section508.22.a

Inspect
Target:#tile-b168635fadf24686a7cbca4a8fdd7bd1 > div:nth-of-type(1) > .tileItem.visualIEFloatFix > .summary-image > img
HTML:

<img src="https://www.york.cuny.edu/library/kanopy-streaming/@@images/image/thumb">;
Summary:

Fix any of the following:

Element does not have an alt attribute
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty or not visible
Element has no title attribute or the title attribute is empty
Element's default semantics were not overridden with role="presentation"
Element's default semantics were not overridden with role="none"

------------------------------------------------------------------------------------------------------------------------------------------------------------
Ensures links have discernible text

wcag2a
wcag111
wcag412
section508
section508.22.a

Inspect
Target:#tile-b168635fadf24686a7cbca4a8fdd7bd1 > div:nth-of-type(1) > .tileItem.visualIEFloatFix > .summary-image
HTML:

<a href="https://www.york.cuny.edu/library/kanopy-streaming"; class="summary-image">
<img src="https://www.york.cuny.edu/library/kanopy-streaming/@@images/image/thumb">;
</a>
Summary:

Fix any of the following:

Element does not have text that is visible to screen readers
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty or not visible
Element's default semantics were not overridden with role="presentation"
Element's default semantics were not overridden with role="none"
Fix all of the following:

Element is in tab order and does not have accessible text

**Comment by vangheem** [comment]
We're doing some issue reaping and closing out inactive issues. Most current project management/bug tracking is being done internally at Wildcard Corp. This tracker will remain open for the public to submit and track issues.

If this bug/feature is still relevant and being worked on/needs fixing, please feel free to reopen.
